views/partials/menu-logo.ejs

```diff
@@ -504,7 +506,8 @@
 			<% } else if (d.type === 'contributors') { %>
 				<a href='/<%- language %>/browse/<%- d.type %>/invited'><%- vocabulary[d.type.slice(0, -1)]?.[language](true)?.capitalize() || d.type.capitalize() %></a>
 			<% } else if (d.type === 'reviews') { %>
-				<a href='/<%- language %>/browse/<%- d.type %>/pending'><%- vocabulary[d.type.slice(0, -1)]?.[language](false, true)?.capitalize() || d.type.capitalize() %></a>
+				<!-- TODO:: TRANSLATE -->
+				<a href='/<%- language %>/browse/<%- d.type %>/pending'><%- ("my reviews")?.capitalize() %></a>
```

Comment:
didn't realize it was using the vocabulary. then please change it in the vocabulary instead.
https://github.com/UNDP-Accelerator-Labs/platform/blob/9a196909e611b54d9b5aa5ad98c6da7913a46a60/views/partials/translations.ejs#L2871-L2920
not sure if `verb` is used anywhere else -- maybe it fits in those cases as well -- otherwise we can add a new parameter for this

(posting here as well)